qdrant-landing/content/documentation/guides/installation.md

```diff
@@ -36,44 +104,79 @@ docker run -p 6333:6333 \
+To change the Qdrant configuration, you can overwrite the production configuration:
 
-Before compiling, make sure that the necessary libraries and the [rust toolchain](https://www.rust-lang.org/tools/install) are installed.
-The current list of required libraries can be found in the [Dockerfile](https://github.com/qdrant/qdrant/blob/master/Dockerfile).
+```bash
+docker run -p 6333:6333 \
+    -v $(pwd)/path/to/data:/qdrant/storage \
+    -v $(pwd)/path/to/custom_config.yaml:/qdrant/config/production.yaml \
+    qdrant/qdrant
+```
 
-Build Qdrant with Cargo:
+Alternatively, you can use your own `custom_config.yaml` configuration file:
 
 ```bash
-cargo build --release --bin qdrant
+docker run -p 6333:6333 \
+    -v $(pwd)/path/to/data:/qdrant/storage \
+    -v $(pwd)/path/to/custom_config.yaml:/qdrant/config/custom_config.yaml \
+    qdrant/qdrant \
+    ./qdrant --config-path config/custom_config.yaml
 ```
 
-After a successful build, the binary is available at `./target/release/qdrant`.
+For more information, see the [Configuration](/documentation/guides/configuration/) documentation.
+
+### Docker Compose
+
+You can also use [Docker Compose](https://docs.docker.com/compose/) to run Qdrant.
+
+Here is an example customized compose file for a single node Qdrant cluster:
+
+```yaml
+services:
+  qdrant:
+    image: qdrant/qdrant:v1.6.1
+    restart: always
+    container_name: qdrant
+    ports:
+      - 6333:6333
+      - 6334:6334
+    expose:
+      - 6333
+      - 6334
+      - 6335
+    configs:
+      - source: qdrant_config
+        target: /qdrant/config/production.yaml
+    volumes:
+      - ./qdrant_data:/qdrant_data
+
+configs:
+  qdrant_config:
+    content: |
+      log_level: INFO  
+```
 
-## Python client
+### From source
 
-In addition to the service itself, Qdrant has a distinct python client, which has some additional features compared to [clients](https://qdrant.tech/documentation/quick_start/#clients) generated from OpenAPI directly.
+Qdrant is written in Rust and can be compiled into a binary executable.
+This installation method can be helpful if you want to compile Qdrant for a specific processor architecture or if you do not want to use Docker.
+
+Before compiling, make sure that the necessary libraries and the [rust toolchain](https://www.rust-lang.org/tools/install) are installed.
+The current list of required libraries can be found in the [Dockerfile](https://github.com/qdrant/qdrant/blob/master/Dockerfile).
 
-To install this client, just run the following command:
+Build Qdrant with Cargo:
 
 ```bash
-pip install qdrant-client
+cargo build --release --bin qdrant
 ```
 
-## Kubernetes
-
-You can use a ready-made [Helm Chart](https://helm.sh/docs/) to run Qdrant in your Kubeternetes cluster.
+After a successful build, the binary is available at `./target/release/qdrant`.
```

Comment:
Missed this... In general, software style guides push "second person," to help readers think "Oh I can do this!"

```suggestion
After a successful build, you can find the binary in the following subdirectory `./target/release/qdrant`.
```